src/renderer/views/Settings/Settings.vue

```diff
@@ -1,33 +1,15 @@
 <template>
   <div>
     <template v-if="unlocked">
-      <general-settings />
-      <hr>
-      <theme-settings />
-      <hr>
-      <player-settings />
-      <hr>
-      <external-player-settings v-if="usingElectron" />
-      <hr v-if="usingElectron">
-      <subscription-settings />
-      <hr>
-      <distraction-settings />
-      <hr>
-      <privacy-settings />
-      <hr>
-      <data-settings />
-      <hr>
-      <proxy-settings v-if="usingElectron" />
-      <hr v-if="usingElectron">
-      <download-settings v-if="usingElectron" />
-      <hr v-if="usingElectron">
-      <parental-control-settings />
-      <hr>
-      <sponsor-block-settings />
-      <hr v-if="usingElectron">
-      <experimental-settings v-if="usingElectron" />
-      <hr>
-      <password-settings />
+      <div
+        v-for="(settingsComponent, i) in sortedSettingsSectionComponents"
+        :key="i"
+      >
+        <hr v-if="i !== 0">
+        <component
+          :is="settingsComponent.type"
+        />
+      </div>
```

Comment:
Please use a `template` element instead of a `div`, that way we don't create div elements that we don't need.

It's always a good idea to use a key that identifies the item if possible, instead of the index, that way Vue can move the existing components around during updates, instead of having to render new ones (notice how the theme settings section stays open when you change the sort order after this change).

```suggestion
      <template
        v-for="(settingsComponent, i) in sortedSettingsSectionComponents"
        :key="settingsComponent.type"
      >
        <hr v-if="i !== 0">
        <component
          :is="settingsComponent.type"
        />
      </template>
```